src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card flat id="name-translations-list">
+  <v-card flat id="name-translations-list" :style="{opacity: isAddingNameTranslation ? '0.4' : '1.0'}">
```

Comment:
neat! I haven't done much `:style` binding myself. 
I don't see anything wrong with this if this is the only place you'd want to apply this. If there was more i'd go with the `:class` binding.

Reply:
yes indeedy, only place.... so far....

Reply:
Sounds good!